arch/x86_64/mcount-insn.c

```diff
@@ -189,24 +197,35 @@ static bool check_unsupported(struct mcount_disasm_engine *disasm, cs_insn *insn
 	return true;
 }
 
-int disasm_check_insns(struct mcount_disasm_engine *disasm,
-		       struct mcount_dynamic_info *mdi, struct sym *sym)
+
+struct mcount_instrument_info *
+disasm_check_insns(struct mcount_disasm_engine *disasm,
+		   struct mcount_dynamic_info *mdi, struct sym *sym)
 {
 	cs_insn *insn = NULL;
-	uint32_t count, i;
+	int status;
+	uint32_t count, i, insns_size;
+	uint8_t insns_byte[MAX_INSNS_BYTES] = {0, };
 	uint8_t endbr64[] = { 0xf3, 0x0f, 0x1e, 0xfa };
-	int ret = INSTRUMENT_FAILED;
 	struct disasm_check_data insn_check = {
 		.addr		= sym->addr + mdi->map->start,
 		.func_size	= sym->size,
 	};
 	struct dynamic_bad_symbol *badsym;
 
+	struct mcount_instrument_info *info =
+		xmalloc(sizeof(struct mcount_instrument_info));
```

Comment:
The convention is to use the variable name like `xmalloc(sizeof(*info))` since it's more flexible to change.

Reply:
And please put a blank line **AFTER** the declaration.

Reply:
thanks for advise, it was changed.